### PR TITLE
fix(cactus-connector-besu): mitigate CVE-2022-24434 and CVE-2022-24999

### LINKED ISSUE #2040: fix(security): vulnerabilities found in cactus-connector-besu
List of vulnerabilities found in cactus-connector-besu image during Azure Container scan.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ZONDER~1.CAL/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ZONDER~1.CAL/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

VULNERABILITY ID | PACKAGE NAME | SEVERITY
-- | -- | --
CVE-2022-24407 | libsasl2-2 | HIGH
CVE-2022-24407 | libsasl2-modules | HIGH
CVE-2022-24407 | libsasl2-modules-db | HIGH
CVE-2021-3711 | libssl1.1 | HIGH
CVE-2022-0778 | libssl1.1 | HIGH
CVE-2021-3711 | openssl | HIGH
CVE-2022-0778 | openssl | HIGH
CVE-2021-3807 | ansi-regex | HIGH
CVE-2021-3807 | ansi-regex | HIGH
CVE-2021-43138 | async | HIGH
CVE-2021-3749 | axios | HIGH
CVE-2022-22143 | convict | HIGH
CVE-2022-21676 | engine.io | HIGH
CVE-2020-8203 | lodash | HIGH
CVE-2021-23337 | lodash | HIGH
CVE-2022-24771 | node-forge | HIGH
CVE-2022-24772 | node-forge | HIGH
CVE-2021-32803 | tar | HIGH
CVE-2021-32804 | tar | HIGH
CVE-2021-37701 | tar | HIGH
CVE-2021-37712 | tar | HIGH
CVE-2021-37713 | tar | HIGH
CVE-2021-23358 | underscore | HIGH

</body>

</html>

### PR DESCRIPTION
**Pull request by aldousalvarez**
Fixes #2040

Signed-off-by: aldousalvarez <aldousss.alvarez@gmail.com>

### CODE REVIEW

**Comment by aldousalvarez** [comment]
Hello Peter, Most of the vulnerabilities are now fixed in cactus-plugin-ledger-connector-besu but there are still some that is not yet fixed as you can see [here](https://github.com/hyperledger/cactus/actions/runs/3728650724/jobs/6323849013
). The only vulnerability (CVE-2022-2421) is still a vulnerability because the latest version of the package that is being used is still the affected version. The changes committed on this PR will fix the 3 out of the 4  remaining vulnerabilities (CVE-2022-24434, CVE-2022-24999, and CVE-2022-24999) once the changes are applied and the new version is released and the packages are updated just like the v1.1 release.

File: .github/containerscan/allowedlist.yaml [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+    - CVE-2022-37734
+    - CVE-2022-25857
 
-    -CVE-2022-37734
-    -CVE-2022-25857
+    #cactus-plugin-ledger-connector-besu
+    - CVE-2022-24434
+    - CVE-2022-24999 #express
+    - CVE-2022-24999 #qs
+    - CVE-2022-2421
```
Comment (petermetz) [inline .github/containerscan/allowedlist.yaml:11]: @aldousalvarez I'd prefer to leave this out, unless there's a specific reason to ignore these CVEs.

**Review by petermetz — CHANGES_REQUESTED** [review]
@aldousalvarez A dozen or so packages have their tests failing so I'm thinking this is probably not a flake but an actual regression. Please fix and then pass it back for review once the tests are passing.

File: .github/containerscan/allowedlist.yaml (hunk 1 above)
Comment (aldousalvarez) [reply .github/containerscan/allowedlist.yaml]: Noted on this, thank you. I already fixed it.

**Review by petermetz — CHANGES_REQUESTED** [review]
@aldousalvarez Same here as with the other vulnerability fix PR Of yours, please clarify the CVE ID in the commit subject and PR title and then we should be good to go.

**Comment by aldousalvarez** [comment]
Hello @petermetz already updated the commit subject and PR title as to your requested changes. Thank you.

**Review by petermetz — APPROVED** [review]
@aldousalvarez Thank you for the updates, LGTM!

**Comment by aldousalvarez** [comment]
Hello @petermetz, After rebasing with the latest commit I have seen that my initial commit was already fixed by the new commits that has been merged so I have no more changes that can be applied. 


Initial Commit
https://github.com/hyperledger/cacti/commit/ce63556c69df7fada460d8972b4ce9f9c1d71a68


Merged Commits:
https://github.com/hyperledger/cacti/commit/d28d5e8f3ed5dff1aa049c8f1bccce79aa21e948
https://github.com/hyperledger/cacti/commit/1cc9667e2432d1d27c2647b86e42707a2e78e4c3

**Pull request closed without merging**